post-processor.go

```diff
@@ -51,6 +52,16 @@ func (p *PostProcessor) Configure(raws ...interface{}) error {
 		return err
 	}
 
+	if p.config.Identifier == "" {
+		return errors.New("Empty `identifier` is not allowed. Please make sure that it is set correctly.")
```

Comment:
__[Golint]__

error strings should not be capitalized or end with punctuation or a newline


<!-- Comment by [Sider](https://sider.review) -->